### PR TITLE
[Fixes #76] use require.resolve, so the lookup is resolved based on n…

### PR DESCRIPTION
**Pull request by stefanpenner**
…ode’s own resolution algorithm

@rwjblue r?

### CODE REVIEW

File: blueprints/ember-cli-eslint/files/.eslintrc.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ var path = require('path');
 
 module.exports = {
   extends: [
-    path.join(__dirname, 'node_modules/ember-cli-eslint/coding-standard/ember-application.js')
+    require.resolve('ember-cli-eslint/coding-standard/ember-application.js')
```
Comment (rwjblue) [inline blueprints/ember-cli-eslint/files/.eslintrc.js]: I'm not 100% if we need the `.js` here (I always have to play with it 😞 ).

Comment (stefanpenner) [inline blueprints/ember-cli-eslint/files/.eslintrc.js]: in node land its the same with or without, unless there is ambiguity `.json` and `.js` with the same basename

**Comment by stefanpenner** [comment]
@rwjblue does this look good?

**Comment by BrianSipple** [comment]
Apologies for not responding on the original PR before this -- but this looks good on my end!

**Comment by rwjblue** [comment]
@stefanpenner - Yep! Looks good to me, I was only curious RE: the extension.

**Comment by stefanpenner** [comment]
@rwjblue any objection to me cutting a minor version bump?

**Comment by rwjblue** [comment]
:shipit:

**Comment by stefanpenner** [comment]
@rwjblue i apparently don't have owner on npm for this project, could you add me?

**Pull request merged**